scripts/bump.sh

```diff
@@ -67,32 +72,38 @@ check_clean_git_working_dir () {
 	[ -n "$DRY_RUN" ] && log_ok "git working directory is clean OK"
 }
 
-# Run all basic checks
-check_pwd
-check_arg "${NB_ARGS}"
-check_dependencies
-check_clean_git_working_dir
-
-# ---------------------------- Variables definition ----------------------------
-
-# Define needed variables
-NEW_VERSION="$1"; shift 1;
-CURRENT_VERSION=$(npm pkg get version | sed 's/"//g')
-BUMP_BRANCH="${NEW_VERSION}-version-bump"
-ISCSC_REMOTE=$(git remote -v | grep 'git@github.com:iScsc/iscsc.fr.git' | awk '{print $1}' | head --lines 1)
-
 # ------------------------------ Advanced checks -------------------------------
 
 # Check that supplied version is semantically correct
 check_version_semantics () {
 	local version="$1"
+	if [ -z "${version}" ]; then
+		log_error "You provided a wrong argument, version can't be computed"
```

Comment:
Maybe "You didn't provide the version" would be a bit more clear than "wrong argument", because a wrong argument can be a number of things

Reply:
Actually with this PR the argument is no longer a version, but a flag! Then if you don't provide either of `-p|-m|-M` `NEW_VERSION` is empty and we have this error 😉

Reply:
Ah yes I forgot that the version was parsed from the argument, it makes sense.